sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -384,6 +386,13 @@ namespace Azure { namespace Core { namespace Http {
      */
     const std::string& GetPath() const { return m_encodedPath; }
 
+    /**
+     * @brief Get the port number set for the url.
```

Comment:
docs consistency nit here/elsewhere - capitalize URL:
```suggestion
     * @brief Get the port number set for the URL.
```